### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by totalschaden**

### CODE REVIEW

**Comment by TooAngel** [comment]
DCBOT: 
This repository is under [democratic collaboration](https://github.com/TooAngel/democratic-collaboration) and will be merged automatically.

The merge decision is based on the outcome of the reviews:
 - `Approve` add the reviewer value (number of commits) to the `metric`
 - `Request changes` substract the reviewer value from the `metric`

 The merge will happen after `(1 - metric/total.votes) * (5 + pull_request.commits * 5))` calculated from the last code change.

Please review the PR to make a good democratic decision.

Summoning some reviewers:
 - @TooAngel: Who are you?
 - @Nevrdid: I just want to get this merged ASAP
 - @WhiteHalmos: Some warm words would be nice

File: README.md [hunk 1]
```diff
@@ -97,3 +97,36 @@ implementation.
...
+
+Game.rooms.W81N49.memory.queue.push is where the creeps is build.
+
+role: is the role the creep will be, you can change this to any available role.
+
+targetRoom: is the creeps target room.
+
+Soon you can also use 
+
+    startAutoSquad(roomfrom,roomto)
```
Comment (Somotaw) [inline README.md]: startAutoSquad('roomFrom' , 'roomTo')

File: README.md [hunk 2]
```diff
@@ -97,3 +97,36 @@ implementation.
...
+targetId: Is in this case the ID of the controller in the target room.
+#### Claiming the Controller (You need a nearby creep with Claim Part/s
+    Game.getObjectById('TheCreepsIdHere').claimController(Game.rooms.RoomNameHere.controller)
+#### Assigning Text to the Controller (readable from Worldmap mouseover, need nearby creep with Claim Part/s)
+    Game.getObjectById('TheCreepsIdHere').signController(Game.rooms.RoomNameHere.controller, "YourTextHere");
+
+• Using the commands above you can also send sourcer, carry, defender etc. to certain rooms/targets.
+
+Soon there will be Squad attacks the Commands for those are:
+#### Send a Squad of Healer and Structurer (only attack structures)
```
Comment (Somotaw) [inline README.md]: +#### Send a Squad of 3 Healers and 3 Structurers (only attack structures,good for walls/ramparts)

File: README.md [hunk 3]
```diff
@@ -97,3 +97,36 @@ implementation.
...
+    Game.getObjectById('TheCreepsIdHere').claimController(Game.rooms.RoomNameHere.controller)
+#### Assigning Text to the Controller (readable from Worldmap mouseover, need nearby creep with Claim Part/s)
+    Game.getObjectById('TheCreepsIdHere').signController(Game.rooms.RoomNameHere.controller, "YourTextHere");
+
+• Using the commands above you can also send sourcer, carry, defender etc. to certain rooms/targets.
+
+Soon there will be Squad attacks the Commands for those are:
+#### Send a Squad of Healer and Structurer (only attack structures) 
+    brain.startSquad('RoomFrom','RoomTo')
+#### Send a Squad of Healer and MeleeAttacker (Attack everything)
```
Comment (Somotaw) [inline README.md]: +#### Send a Squad of 3 Healers and 1 MeleeAttacker (Attack everything)

File: README.md (hunk 1 above)
Comment (TooAngel) [inline README.md]: I use usually all capital to make clear that this needs to be replaced.

**Comment by TooAngel** [comment]
DCBOT: A new review, yeah.

    Votes: 126/296
    Coefficient: 0.425675675676
    Merging in 5 days -5 hours
    Age 0 days 22 hours

**Comment by Somotaw** [comment]
@totalschaden

**Comment by democratic-collaboration** [comment]
DCBOT: A new review, yeah.

    Votes: 131/302
    Coefficient: 0.433774834437
    Merging in 9 days -15 hours
    Age 2 days 21 hours

**Comment by democratic-collaboration** [comment]
DCBOT: A new review, yeah.

    Votes: 127/306
    Coefficient: 0.415032679739
    Merging in 8 days -3 hours
    Age 3 days 18 hours

**Pull request merged**